docs/getting_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## What is the goal of Inago?
+
+Inago is a deployment tool that manages groups of unit files to deploy them to
+a fleet cluster similar to `fleetctl`. Since `fleetctl` is quite limited, Inago
+aims to abstract units away and provide more sugar on top like update
+strategies. That way the user can manage unit files more easily.
+
+## Prerequisites
+
+Inago requires a certain directory structure and unit file names to make the
+tool actually work. There must be a group folder with unit files in it. For details see the [Unit File Structure](structure.md) chapter.
+
+## Working with Inago
+
+The basic commands you can use with Inago are `submit`, `start`, `stop`, `destroy`, and `status`. The subject of your commands is always a unit group as defined by a group folder.
+
+### Submit
+
+Submitting groups enables deploying multiple slices using one command.
+
+```
+inagoctl submit myapp 3
+```
+
+This assumes there is a directory `myapp` and that the unit files in that directory cary an `@` and thus are to be sliced. The argument `3` tells Inago to create 3 slices of the units found in the group directoy. Inago looks up the
```

Comment:
is "carry" the right word here? I'd go with something like "the unit files in that directory have `@` in their name, and can therefore be sliced"